NEWS.md

```diff
@@ -1,4 +1,4 @@
-## 1.7.0 IN_PROGRESS
+## 1.7.0 2022-10-18
```

Comment:
The release version should be 1.6.1 because all changes since 1.6.0 are dependency upgrade that only bump the patch version:
* Vert.x from 4.3.1 to 4.3.4
* Spring from 5.3.20 to 5.3.23

FOLIO uses semantic versioning:
https://dev.folio.org/guidelines/contributing/#implementation-versions

Reply:
Maven plugins execute at build time only. Bumping a plugin version doesn't influence the runtime jar and the runtime API.

The folio-di-support version number is an indication for the users of folio-di-support and is about the runtime jar only.

Therefore 1.6.1 seems more appropriate to me.